helpdesk-dynamic-rendering/webpack.config.js

```diff
@@ -59,5 +59,15 @@ module.exports = {
     new webpack.optimize.DedupePlugin(),
     new webpack.optimize.UglifyJsPlugin(),
     new webpack.optimize.AggressiveMergingPlugin()
-  ]
+  ],
+  devServer: {
```

Comment:
Please update README file to include how to run the application using webpack dev server.

Reply:
done